### DISCUSSION TITLE
How to remove the Systemd prompt

### DISCUSSION THREAD

**Discussion by dhirschfeld** [starship/starship#3611] [Q&A] [answered]
I just deployed a new container and am seeing a Systemd in front of my prompt:

![image](https://user-images.githubusercontent.com/881019/153867270-e865c36b-31fc-4a36-aed4-d88c5ef652cc.png)

I don't like it, how can I remove it?

**Comment by dhirschfeld** [comment]
```
⬢ [Systemd] ❯ starship --version
starship 1.3.0
tag:v1.3.0
commit_hash:e6a921b5
build_time:2022-02-07 15:03:09 +00:00
build_env:rustc 1.58.1 (db9d1b20b 2022-01-20),
```

**Comment by davidkna** [comment, marked as answer]
You should be able to remove this from the prompt by running `starship config container.disabled true`.

**Comment by dhirschfeld** [comment]
Thanks @davidkna - I'll give it a go and report back...

**Comment by dhirschfeld** [comment]
Adding the below in my [`starship.toml`](https://starship.rs/config/#container) config worked great - thanks @davidkna! 
```toml
[container]
    disabled = true
```
